### PR TITLE
Admin notification about new users

### PR DESCRIPTION
**Pull request by Axedez**
[Trello Task](https://trello.com/c/fEGNgpm0/51-admin-notification-when-user-is-registered)

**CHECK LIST**
- [x] specs added
- [x] specs passed
- [x] code coverage >90%
- [ ] DB indexes added if needed
- [ ] seed.rb file is updated if needed
- [x] PR is reviewed manually again (to make sure you have 100% ready code)
- [x] PR doesn't have conflicts with main branch
- [ ] I've checked new feature as logged in and logged out user if needed
- [x] "let!" is used in specs
- [x] screenshot of feature is added if task has UI
image
image

### CODE REVIEW

File: app/views/admin_mailer/admin_notification.html.slim [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    tr
+      td
+        = t('devise.mailer.admin_notification.user_phone')
+        |:
+      td = @user.mobile
+    tr
+      td
+        = t('devise.mailer.admin_notification.created_at')
+        |:
+      td = @user.created_at
```
Comment (mmarusyk) [inline app/views/admin_mailer/admin_notification.html.slim:34]: Make readable date

File: app/models/user.rb [hunk 2]
```diff
@@ -31,15 +31,23 @@ class User < ApplicationRecord
...
 
+  def send_mail_to_admin
+    AdminMailer.admin_notification(self).deliver_now
+  end
+
   def full_name
     "#{first_name} #{last_name}"
   end
+
+  def self.admin
```
Comment (pereslop) [inline app/models/user.rb:48]: No need separate method, read about rails scopes

File: spec/mailers/admin_mailer_spec.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+RSpec.describe AdminMailer, type: :mailer do
+  describe "notify" do
+    let(:user) { FactoryBot.create(:user, role: :admin) }
+    let(:mail) { AdminMailer.admin_notification(user) }
```
Comment (pereslop) [inline spec/mailers/admin_mailer_spec.rb:6]: Let!

File: spec/mailers/admin_mailer_spec.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+    it "renders the headers" do
+      expect(mail.subject).to eq("New user was registered!")
```
Comment (pereslop) [inline spec/mailers/admin_mailer_spec.rb:9]: Use i18n

Comment (Axedez) [reply spec/mailers/admin_mailer_spec.rb:9]: There is no need, because subject was only in English

**Review by Ukhanskyi — APPROVED** [review]

**Review by mmarusyk — APPROVED** [review]

**Review by DenisRem — APPROVED** [review]

File: app/models/user.rb [hunk 5]
```diff
@@ -40,6 +44,8 @@ def send_welcome_email_to_new_user
   def full_name
     "#{first_name} #{last_name}"
   end
+
+  scope :is_admin, -> { where(role: ADMIN) }
```
Comment (chu4) [inline app/models/user.rb:48]: no need for this scope
remember u have enum for roles
https://api.rubyonrails.org/v5.2.3/classes/ActiveRecord/Enum.html

**Review by chu4 — APPROVED** [review]

**Pull request merged**